cmd/kconf.go

```diff
@@ -141,6 +141,49 @@ var viewCmd = &cobra.Command{
 	},
 }
 
+var useCmd = &cobra.Command{
+	Use:   "use",
+	Short: "set current context",
+	Long:  `set current context by user provide the name`,
```

Comment:
`set the current context in the main kubeconfig`

Reply:
OK, I will change it.